spec/support/spec/ipset.rb

```diff
@@ -36,4 +36,24 @@
       ],
     )
   end
+
+  it do
+    is_expected.to contain_ipset__set('dev').with_set(
+      %w[
```

Comment:
I had a discussion recently with another dev about unit tests. There isn't much logic around for the ipsets. You define it in hiera and the resources will be created like that. Is it worth it writing tests for that? I don't know. I often write them to feeld good, but I don't think they provide much value. Another idea that might bring more value: count how many ipset resources you have in the catalog (like documented here: https://rspec-puppet.com/documentation/classes/#testing-the-number-of-resources-of-a-specific-type).

Reply:
So the reason I do that is basically double entry accounting.  I don't cut'n'paste the values from hiera but instead re-key them or cut'n'paste from the source of truth as a way of making sure the hiera value is actually correct.  It has also been helpful during refactors to catch when a class is accidentally dropped from the manifest. And it makes me feel better too... I would very much prefer to do this in rspec-beaker...

Reply:
yes. beaker would be cool. But this is a good check to catch wrong typos in the data.